packages/big-design/src/managers/alerts/spec.tsx

```diff
@@ -0,0 +1,114 @@
+import { render } from '@test/utils';
+import '@testing-library/react';
+import 'jest-styled-components';
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+
+import { AlertProps } from '../../components/Alert';
+
+import { createAlertsManager } from './manager';
+import { AlertsManager } from './AlertsManager';
+
+let alertsManager: ReturnType<typeof createAlertsManager>;
+
+beforeEach(() => {
+  alertsManager = createAlertsManager();
+});
+
+describe('alertsManager', () => {
+  const testKey = 'test';
+  const alert = ({
+    text: 'Test',
+  } as unknown) as AlertProps;
+
+  test('adds a message', () => {
```

Comment:
Can we change the description or the assertion? If it returns a key then it means it added a message? 🤔